deploy/redis-writer.yaml

```diff
@@ -0,0 +1,253 @@
+# Copyright 2025 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: redis-writer
+objects:
```

Comment:
why don't you add the valkey-writer as a new object inside the cache-writer.yaml? that way you will be able to reuse all the parameters + no need to change app-interface

Reply:
That will be much better, will do :)